src/main/java/mate/academy/springboot/swagger/repository/ProductRepository.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.springboot.swagger.repository;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product, Long> {
+    @Query("from Product p where p.price between ?1 and ?2")
+    List<Product> getAllByPriceBetween(BigDecimal from, BigDecimal to, PageRequest pageRequest);
```

Comment:
please, check if method works without `Query` annotation 
when repository extends from `PagingAndSortingRepository<Product, Long>` 
(if not, try also to name it this way: `findAllByPriceBetween`)